content/docs/querying/functions.md

```diff
@@ -89,7 +89,8 @@ month for each of the given times in UTC. Returned values are from 28 to 31.
 `delta(v range-vector)` calculates the difference between the
 first and last value of each time series element in a range vector `v`,
 returning an instant vector with the given deltas and equivalent labels.
-The delta is interpolated to cover the full time range.
+The delta is interpolated to cover the full time range, so that it is 
```

Comment:
interpolated -> extrapolated

full time range -> full time range as specified in the range vector selector